.github/workflows/release-from-main.yml

```diff
@@ -1,33 +1,35 @@
-name: Create Release from main
+name: Create releases from main every day
 on:
-  push:
-    branches:
-      - main
+  schedule:
+    - cron: "00 00 * * *" # Every day at 00:00 UTC
```

Comment:
@AgustinRamiroDiaz, it would be better to release at 8 a.m. UTC, so we have the entire day in Europe and America to fix anything. If we release at midnight, there would be several hours without anyone capable of assisting any user.

Reply:
fixed!

Reply:
I think that you left the comment unchanged :)